src/lib.rs

```diff
@@ -1,30 +1,52 @@
+/// # Fantom libhash-sha3
```

Comment:
Module doc comment literal is `//!`, not `///`.

Please check your results with `cargo doc`.

I think if you provide a title with a hash, it should be descriptive. It's OK not to have a title too, for example: https://rust-lang-nursery.github.io/futures-api-docs/0.3.0-alpha.17/src/futures/lib.rs.html#1-534